### PR TITLE
add option to disable google consent mode

### PR DESCRIPTION
**Pull request by mrbatista**
See "Script tag data attributes" section on https://www.cookiebot.com/en/developer/

### CODE REVIEW

File: projects/ngx-cookiebot/src/lib/ngx-cookiebot.config.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@
  */
 export abstract class NgxCookiebotConfig {
   blockingMode: 'auto' | 'manual' | string;
+  consentMode?: string;
```
Comment (erichjsonfosse) [inline projects/ngx-cookiebot/src/lib/ngx-cookiebot.config.ts:6]: Could we be more specific here? The documentation seems to indicate that `disabled` is the only valid value to pass, no?

**Review by erichjsonfosse — CHANGES_REQUESTED** [review]
Many thanks for you contribution, @mrbatista !
Would you consider adding a short description of usage to the readme file as well (the one in the project folder)?

File: projects/ngx-cookiebot/src/lib/ngx-cookiebot.config.ts (hunk 1 above)
Comment (mrbatista) [reply projects/ngx-cookiebot/src/lib/ngx-cookiebot.config.ts:6]: Yes, it makes sense to check the allowed values.

**Comment by mrbatista** [comment]
@erichjsonfosse all changes are done.

**Review by erichjsonfosse — APPROVED** [review]

**Pull request merged**